### DISCUSSION TITLE
【求助】如何在一小时内睡够八个小时

### DISCUSSION THREAD

**Discussion by yigexie** [share121/inter-knot#2718] [其他]
本人还有一小时将要去学校，但是本人现在非常的困，想要一觉睡到明天早上，故发此帖，求助绳网
<img src="https://img-static.mihoyo.com/communityweb/upload/ee7bb800b0c90f66fa9e321cb6d8ee17.png" alt="抱大腿" width="75" >

**Comment by Xmug** [comment]
个人经验：弄一个别扭的姿势睡下，心中不断告诉自己一定要及时醒来，醒来之后能够保证两小时内的清醒。

**Comment by fa-e-tong-LKN** [comment]
所以楼主现在是不是已经到学校了<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by MrDucker** [comment]
丸辣！<img src="https://bbs-static.miyoushe.com/static/2024/03/21/3248c770e7e0e7ec4a167605872b6d66_1511725123905663480.png" alt="丽都漫步-格莉丝" width="75" >

**Comment by MrDucker** [comment]
丸辣！<img src="https://bbs-static.miyoushe.com/static/2024/03/21/3248c770e7e0e7ec4a167605872b6d66_1511725123905663480.png" alt="丽都漫步-格莉丝" width="75" >

**Comment by JOYI147** [comment]


**Reply by Nie7bai** [reply]
添柴！

**Comment by AnkaAlioth** [comment]
据说20分钟冥想=2小时睡眠，你要不是试试？<img src="https://img-static.mihoyo.com/communityweb/upload/53eceb38cca6c5c3cf12c838572c651b.png" alt="阿波尼亚_今天周一" width="75" >

**Comment by Kujojolen** [comment]
花一小时做法，让时间静止八个小时就行

**Comment by Vgvn** [comment]
楼主再见

**Comment by blacktree-eme** [comment]
当你看到我的时候，已经晚辣

**Comment by SKAIZH** [comment]
推荐b站up闪老师

**Comment by demolute** [comment]
只要你强迫世界整体高速运动，而你静止。世界运动速度够快的情况下，世界过去了一小时，你这边就已经过去了八小时

**Comment by aizai1173** [comment]
我有个主意，到学校再睡吧<img src="https://img-static.mihoyo.com/communityweb/upload/d41b52daa44ef813710cf4b482a256b2.png" alt="三月七_点赞" width="75" >

**Comment by CJHXAMSTLP** [comment]
很简单，你睡觉时光速飞行就是了<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by LuAn-afl** [comment]
这个简单只要你以超过光速8小时的速度奔跑你就能回到8小时前了睡一觉了<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by homteam** [comment]
[【干货】如何3小时睡够8小时？]( https://www.bilibili.com/video/BV12U411Z7A1/?share_source=copy_web&vd_source=1b9c9ba08818c4515231fa3984e88d43)<img src="https://bbs-static.miyoushe.com/static/2024/06/12/580c5fe822ac37260b8c95decbd5bde6_4418994548529724081.png" alt="希格雯-护肤" width="75" >

**Reply by silverwolf0000** [reply]
我就知道肯定会有人附上这个链接

**Comment by yivitte** [comment]
直接睡八个小时，然后等着挨骂就行了

**Comment by opqphp** [comment]
一路走好<img src="https://upload-bbs.mihoyo.com/upload/2022/11/11/db3b5981feb2c2076c8582ed0f05b2dc_4253757706889588004.png" alt="胡桃-在路上了" width="75" >
